processor/routingprocessor/factory_test.go

```diff
@@ -16,6 +16,13 @@ package routingprocessor
 
 import (
 	"context"
+	"go.opentelemetry.io/collector/component"
```

Comment:
Imports order needs fixing. Just run `make gofmt` and it should be good then

Reply:
`make gofmt` has been ran and changes pushed